data/json/npcs/robofac/robofac_intercom/robofac_intercom_trades.json

```diff
@@ -232,5 +232,235 @@
       },
       { "text": "I'll have to give it some thought.", "topic": "TALK_ROBOFAC_INTERCOM_SERVICES" }
     ]
+  },
+  {
+    "id": [
+      "TALK_ROBOFAC_INTERCOM_SELL_YRAX_TOYS",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIFACET",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIAKIS",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_GOLDEN_MONOLITH",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_DELTA",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT_LEG",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIFACET",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIAKIS",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_GOLDEN_MONOLITH",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_DELTA"
+    ],
+    "type": "talk_topic",
+    "dynamic_line": "Beautiful; please put them into loading area.  We'll issue a reward based depending on what you managed to scavange.",
+    "responses": [
+      {
+        "text": "[Put in broken trifacet.]",
+        "condition": { "u_has_items": { "item": "broken_yrax_trifacet", "count": 1 } },
+        "effect": [
+          { "u_consume_item": "broken_yrax_trifacet", "count": 1, "popup": true },
+          { "math": [ "u_broken_yrax_trifacet_sold", "++" ] },
+          { "u_spawn_item": "RobofacCoin", "count": 1 }
+        ],
+        "topic": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIFACET"
+      },
+      {
+        "text": "[Put in broken triakias.]",
+        "condition": { "u_has_items": { "item": "broken_yrax_triakis", "count": 1 } },
+        "effect": [
+          { "u_consume_item": "broken_yrax_triakis", "count": 1, "popup": true },
+          { "math": [ "u_broken_yrax_triakis_sold", "++" ] },
+          { "u_spend_cash": -1000 }
```

Comment:
Any reason why sometimes credit is given directly whereas other times they give a physical token of their currency instead? Seems inconsistent.

Reply:
credit is given for broken/ small robots ; real tokens are quite expensive

Reply:
Do I understand correctly that -1000 = -10$ (which is in turn <1 Hub coin)? (Where $ is this nebulous post-apocalypse "dollar".) If so, that seems reasonable since the Hub does not issue "Hub cents".

Reply:
Yes